src/common/k8s-api/__tests__/kube-api.test.ts

```diff
@@ -304,19 +304,28 @@ describe("KubeApi", () => {
 
   describe("watch", () => {
     let api: TestKubeApi;
+    let stream: PassThrough;
 
     beforeEach(() => {
       api = new TestKubeApi({
         request,
         objectConstructor: TestKubeObject,
       });
+      stream = new PassThrough();
```

Comment:
All these `stream` changes is to get rid of those annoying logger.error calls during the unit tests.